### DISCUSSION TITLE
Maintaining dev conformance up to AAA

### DISCUSSION THREAD

**Discussion by royemosby** [nditcommunity/ndit-website#43] [Accessibility]
We aim to achieve AAA conformance. In cases where some feature decreases the level of the site, all considerations will be taken to determine if the tradeoff is worth it.

Accessibility considerations start with design but we also need checks in place for future development.

We will do this initially with manual checks. Some tools include:

- [Chrome's Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/)
- [Deque's axe DevTools](https://www.deque.com/axe/devtools/) - large costs for full functionality
- [WAVE Extension](https://wave.webaim.org/extension/)
- [WAVE API](https://wave.webaim.org/api/)

Eventually, I would like to implement a linter check prior to build.

**Comment by hayleyw7** [comment]
Thanks! Just so we have it here somewhere if it's not yet, I'll link to the accessibility ticket where there's also some links that came up.

https://github.com/nditcommunity/ndit-website/issues/35
